src/groovy/org/grails/rabbitmq/RabbitServiceConfigurer.groovy

```diff
@@ -216,6 +217,10 @@ class RabbitServiceConfigurer {
             connectionFactory = ref("rabbitMQConnectionFactory")
             concurrentConsumers = serviceConcurrentConsumers
             messageListener = ref("${propertyName}RabbitAdapter")
+
+            if (type != SUBSCRIBE && serviceGrailsClass.clazz.name == 'skillz.TournamentConcludeService') {
```

Comment:
Why do we care about type? Do you know what this value is generally set to?

Reply:
It is normally set to `rabbitQueue`